### DISCUSSION TITLE
about imodeljs version

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#3245] [Q&A] [answered]
the github imodeljs publish the version  : release/2.19.32   3 days ago and  release/3.0.0 28 days ago ,
 imodejs will developing on 2.0?

**Comment by pmconne** [comment, marked as answer]
We produce new versions of 2.x for critical bug fixes only. All new feature development occurs on 3.x.

**Reply by paprophet** [reply]
ok  thank you !
